src/main/java/com/google/googleidentity/oauth2/util/OAuth2Utils.java

```diff
@@ -41,4 +46,51 @@
         return ImmutableSet.copyOf(scopes);
 
     }
+
+    /**
+     * Get UserSession from HttpSession
+     *
+     * @param request
+     * @return
```

Comment:
Since the @param and @return label don't provide any extra information and the parameter name is clear enough, you can delete these two lines. Same for other methods/classes.